README.md

```diff
@@ -36,6 +38,41 @@ func main() {
 }
 ```
 
+#### HandlerFunc
+```go
+package main
+
+import (
+  "net/http"
+  "time"
+
+  "github.com/hellofresh/health-go"
+  "github.com/pressly/chi"
+)
+
+func main() {
+  health.Register(health.Config{
```

Comment:
At least in my vim it auto format once I save the file, and normally I don't see 8 spaces on readme normally here is another example of 2 tabs indentation https://github.com/segmentio/kafka-go#reader